bindata/network/ovn-kubernetes/008-ovs-flows-config.yaml

```diff
@@ -0,0 +1,54 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ovs-flows-config-publisher
```

Comment:
So let me preface this by saying i'm not an RBAC expert...... However, I've read though https://docs.google.com/document/d/1AeUgETCtsVsYPuvpu0HuDfeuhTaWGRd3Qpkf38bu81U/edit#heading=h.1vsfycjyceaq and tend to think that the CNO should only be defining RBAC roles that it intends to use (for resources it doesn't own), or delegate out(for resources it does own) 

In this case we're defining an a role in the CNO for an resource it doesn't create, nor will it ever use the role.  

I _think_  we should instead have the NOO make/ bind the `ovs-flows-config-publisher` role

Reply:
Ok, sounds reasonable. I'll remove this